### PR TITLE
depth chart ACs

### PR DESCRIPTION
**Pull request by daro-maj**

### CODE REVIEW

File: user-interface/6006-DEPC-chart_depth.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Depth chart
+
+As a market user, I want to be aware of the depth of the market using a chart.
+
+- **Must** must see mid market price in the center of chart (<a name="6006-DEPC-001" href="#6006-DEPC-001">6006-DEPC-001</a>)
+- **Must** must see green bid wall and vertical axis on the left side (<a name="6006-DEPC-002" href="#6006-DEPC-002">6006-DEPC-002</a>)
+- **Must** must see red ask wall and vertical axis on the right side (<a name="6006-DEPC-003" href="#6006-DEPC-003">6006-DEPC-003</a>)
+- **Must** must see X-axis - buy or sell price (<a name="6006-DEPC-004" href="#6006-DEPC-004">6006-DEPC-004</a>)
+- **Must** must see Y-axis - value (<a name="6006-DEPC-005" href="#6006-DEPC-005">6006-DEPC-005</a>)
```
Comment (bwallacee) [inline user-interface/6006-DEPC-chart_depth.md]: is value the correct term here?

Is it the number of orders?

File: user-interface/6006-DEPC-chart_depth.md
Comment (bwallacee) [inline user-interface/6006-DEPC-chart_depth.md:1]: Not sure how detailed we want to go with this but i have a few other ideas for potential ACS:

- On mouse hover opposite ask or bid price is shown
- On mouse hover, graph lines are shown
- Can "zoom" in

Not one that we can cover now, but with market-sim, would we want an AC around the depth chart updating as new bids/asks come onto the book?

File: user-interface/6006-DEPC-chart_depth.md (hunk 1 above)
Comment (daro-maj) [reply user-interface/6006-DEPC-chart_depth.md]: this is amount of currency

File: user-interface/6006-DEPC-chart_depth.md
Comment (daro-maj) [reply user-interface/6006-DEPC-chart_depth.md:1]: 1. Good point 
2. It is here must see red ask wall and _vertical axis on the right side_ but I can split 
3.. It is here  **Must** must be able to zoom in and zoom out chart

File: user-interface/6006-DEPC-chart_depth.md (hunk 1 above)
Comment (daro-maj) [reply user-interface/6006-DEPC-chart_depth.md]: I have changed to  cumulative volume

**Review by bwallacee — APPROVED** [review]

**Pull request merged**